gemini/master.py

```diff
@@ -13,11 +15,48 @@
 NODE_TOKEN_BYTES = 32
 NODE_PASSWORD_BYTES = 32
 
+STORAGE_TYPE = "memory"
+
 app = Flask('gemini-master')
 
 c = consul.Consul(host='127.0.0.1', port=8500)
 
-jobs = []
+jobs: List[Any] = []
+
+
+class MemoryNodeInfoRepository:
+    def __init__(self):
+        self.node_data = {}
+
+    def put_data(self, value: dict):
+        self.node_data[value['node_id']] = value
+
+    def get_data(self, node_id: str) -> dict:
+        return self.node_data[node_id]
+
+
+class ConsulNodeInfoRepository:
```

Comment:
`MemoryNodeInfoRepository`  and `ConsulNodeInfoRepository` should implement the same abstract base class and the common methods should be marked as abstract methods. 

See: https://docs.python.org/3/library/abc.html